### PR TITLE
fix: remove link panel from affine-doc-editor

### PR DESCRIPTION
**Pull request by regischen**
\cc @pengx17

### CODE REVIEW

File: packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts [hunk 1]
```diff
@@ -201,16 +201,13 @@ export class BiDirectionalLinkPanel extends WithDisposable(LitElement) {
   }
 
   private get _docPageElement() {
-    const affineDocEditor = this.closest('affine-doc-editor');
-    assertExists(affineDocEditor);
-    const docPageElement = affineDocEditor.querySelector('affine-doc-page');
+    const docPageElement = this.parentElement?.querySelector('affine-doc-page');
```
Comment (pengx17) [inline packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts:204]: The parent element is not necessarily the one that wraps both `affine-doc-page` & `bi-directional-link`

Comment (pengx17) [reply packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts:204]: I think you can add a new property to let the upstream to pass in an doc-page element instead

Comment (regischen) [reply packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts:204]: Alright, that sounds reasonable.

Comment (regischen) [reply packages/presets/src/fragments/bi-directional-link/bi-directional-link-panel.ts]: done

**Review by pengx17 — APPROVED** [review]

**Review by doodlewind — APPROVED** [review]

**Pull request merged**